lightning/src/ln/channelmanager.rs

```diff
@@ -3134,6 +3134,9 @@ where
 				PendingOutboundPayment::InvoiceReceived { .. } => {
 					Some(RecentPaymentDetails::AwaitingInvoice { payment_id: *payment_id })
 				},
+				PendingOutboundPayment::StaticInvoiceReceived { .. } => {
+					Some(RecentPaymentDetails::AwaitingInvoice { payment_id: *payment_id })
```

Comment:
Since we may be in the `StaticInvoiceReceived` state for a long time, may want to add a new `RecentPaymentDetails` variant for this, at least once we stop cfg-gating.